tests/Fixer/Import/GroupImportFixerTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Import;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Vladimir Kuprienko <vldmr.kuprienko@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Import\GroupImportFixer
+ */
+final class GroupImportFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return [
+            [
+                '<?php
+
+namespace Test;
+
+use Foo\{Bar, Baz, Test};
+',
+                '<?php
+
+namespace Test;
+
+use Foo\Bar;
+use Foo\Baz;
+use Foo\Test;
+',
+            ],
+            [
+                '<?php
+use App\First;
+use Test\Second;
+use Foo\{Bar, Baz};
+',
+                '<?php
+use App\First;
+use Foo\Bar;
+use Test\Second;
+use Foo\Baz;
+',
+            ],
+            [
+                '<?php
+use App\{First, Second};
+use Foo\{Bar, Baz};
+',
+                '<?php
+use Foo\Bar;
+use Foo\Baz;
+use App\First;
+use App\Second;
+',
+            ],
+            [
+                '<?php
+use App\{First, Second};
+use Foo\{Bar, Baz};
+',
+                '<?php
+use Foo\Bar;
+use App\First;
+use Foo\Baz;
+use App\Second;
+',
+            ],
+            [
+                '<?php
+use Foo\{Bar as Test, Baz};
+use App;
+',
+                '<?php
+use Foo\Bar as Test;
+use Foo\Baz;
+use App;
+',
+            ],
+            [
+                '<?php
+use App\Repository\{Customer as Client, Profile, User};
+',
+                '<?php
+use App\Repository\User;
+use App\Repository\Profile;
+use App\Repository\Customer as Client;
+',
+            ],
+            [
+                '<?php
+use function Foo\{Bar, Baz, Test as Alias};
+',
+                '<?php
+use function Foo\Bar;
+use function Foo\Baz;
+use function Foo\Test as Alias;
+',
+            ],
+            [
+                '<?php
+use const Some\Place\{A, B, C as D};
+',
+                '<?php
+use const Some\Place\A;
+use const Some\Place\B;
+use const Some\Place\C as D;
+',
+            ],
+        ];
```

Comment:
Please add tests for comments in between tokens, make sure you don't throw any comments away

Reply:
@ntzm I'll add tests fot comments. Thanks!

Reply:
I meant something like this:

```php
/*1*/use/*2*/A/*3*/\/*4*/B/*5*/;/*6*/
/*7*/use/*8*/A/*9*/\/*10*/C/*11*/;/*12*/
/*13*/use/*14*/A/*15*/\/*16*/D/*17*/;/*18*/
```

;)

Reply:
@ntzm ok, updated.

Reply:
Brilliant, thanks